ncm-cron/src/main/perl/cron.pm

```diff
@@ -453,76 +510,79 @@ sub cleanCrontabs {
             # Not yet implemented
             next;
         } else { # Linux
-            my $fname = "$name.ncm-cron.cron";;
+            my $fname = "$name$CRON_ENTRY_EXTENSION";;
             $fname =~ s{[/\s]}{_}g;
             $cronEntryNames{$fname} = ''; # Value is meaningless
         }
     }
 
-    # Solaris: Collect the crontabs in /var/spool/cron/crontabs then delete
-    # any ncm-cron entries. These are identified as lines between:
-    #     NCM-CRON START:
-    #     NCM-CRON END:
+    my $crondir = $osname eq "solaris" ? $CRONDIR_SOLARIS : $CRONDIR_LINUX;
+    my $all_files = $self->listdir($crondir,
```

Comment:
hmmm, note to self: this is ugly, probably deserves it's own option (listing a dir and returning all files sounds very common for cfgmgt toolkit)